daniel-davidF/styles/modules.css

```diff
@@ -0,0 +1,70 @@
+.site-header {
+  background: #FFEAAA;
+}
+
+/* REVIEW: Float the main header to the left, to make room for the hamburger menu. */
+.heading-group-h1 {
+  float: left;
+  padding: 0 0 .2em 1em;
+}
+
+/* DONE: Add styles for the icon-menu class. It should be bigger, and off to the right. */
+.icon-menu{
+ float: right;
+ display:inline;
+}
+
+.main-nav {
+  padding: 0.1em;
+}
+
+.main-nav ul {
+  display: none;
+  padding: 0;
+  list-style: none;
+  clear: both;
+}
+
+.main-nav:hover ul {
+  display: block;
+}
+
+.main-nav li {
+  margin-right: .4em;
+  font-size: 1.5em;
+  padding: .5em .2em;
+  border-bottom: 1px solid #ccc;
+  display: inline;
+}
+
+#filters {
+  padding: 0;
+  list-style: none;
+}
+
+.template {
+  display: none;
+}
+
+.byline {
+  color: #777;
+}
+
+.byline a {
+  font-weight: bold;
+  text-decoration: none;
+}
+
+/* DONE: Now add styles for how things should look on a larger screen:
+         - Hide the hamburger.
+         - Main nav should be all on one row, with padding spaced better for a larger screen.
```

Comment:
The main nav does not show up when the screen is large with your current styling.